### PR TITLE
[BrowserEngineKit] Implement Xcode 16.2 beta 1-3 changes.

### PR DESCRIPTION
**Pull request by rolfbjarne**

### CODE REVIEW

File: src/browserenginekit.cs [hunk 1]
```diff
@@ -1194,5 +1222,134 @@ public enum BEAccessibilityNotification : long {
 		[Field ("BEAccessibilityValueChangedNotification")]
 		Changed,
 	}
+
+	[iOS (18, 2), TV (18, 2), Mac (15, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (NSObject))]
+	interface BEAccessibilityTextMarker : INSCopying, INSSecureCoding
```
Comment (dalexsoto) [inline src/browserenginekit.cs:1228]: Shouldn’t this be using the `I` less version of the protocol names? Or did we fix that?

```suggestion
	interface BEAccessibilityTextMarker : NSCopying, NSSecureCoding
```

File: src/browserenginekit.cs [hunk 2]
```diff
@@ -1194,5 +1222,134 @@ public enum BEAccessibilityNotification : long {
...
+	{
+	}
+
+	[iOS (18, 2), TV (18, 2), Mac (15, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (NSObject))]
+	interface BEAccessibilityTextMarkerRange : INSCopying, INSSecureCoding
```
Comment (dalexsoto) [inline src/browserenginekit.cs:1234]: Same

```suggestion
	interface BEAccessibilityTextMarkerRange : NSCopying, NSSecureCoding
```

File: src/browserenginekit.cs [hunk 3]
```diff
@@ -1194,5 +1222,134 @@ public enum BEAccessibilityNotification : long {
...
+		[Export ("startMarker", ArgumentSemantic.Strong)]
+		BEAccessibilityTextMarker StartMarker { get; set; }
+
+		[Export ("endMarker", ArgumentSemantic.Strong)]
+		BEAccessibilityTextMarker EndMarker { get; set; }
+	}
+
+	[iOS (18, 2), TV (18, 2), Mac (15, 2), MacCatalyst (18, 2)]
+	[Protocol]
+	interface BEAccessibilityTextMarkerSupport
```
Comment (dalexsoto) [inline src/browserenginekit.cs:1245]: Any particular reason not to use new style protocol support?

File: src/browserenginekit.cs [hunk 4]
```diff
@@ -1194,5 +1222,134 @@ public enum BEAccessibilityNotification : long {
...
+	delegate void BEDownloadMonitorBeginMonitoringCallback ([NullAllowed] BEDownloadMonitorLocation placeholderLocation, [NullAllowed] NSError error);
+	delegate void BEDownloadMonitorResumeMonitoringCallback ([NullAllowed] NSError error);
+
+	[NoTV, NoMac, iOS (18, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface BEDownloadMonitor
+	{
+		[Export ("initWithSourceURL:destinationURL:observedProgress:liveActivityAccessToken:")]
+		NativeHandle Constructor (NSUrl sourceUrk, NSUrl destinationUrl, NSProgress observedProgress, NSData liveActivityAccessToken);
```
Comment (dalexsoto) [inline src/browserenginekit.cs:1327]: Typo

```suggestion
		NativeHandle Constructor (NSUrl sourceUrl, NSUrl destinationUrl, NSProgress observedProgress, NSData liveActivityAccessToken);
```

**Review by dalexsoto — CHANGES_REQUESTED** [review]

File: src/browserenginekit.cs [hunk 5]
```diff
@@ -1194,5 +1222,134 @@ public enum BEAccessibilityNotification : long {
...
+	{
+		[Export ("startMarker", ArgumentSemantic.Strong)]
+		BEAccessibilityTextMarker StartMarker { get; set; }
+
+		[Export ("endMarker", ArgumentSemantic.Strong)]
+		BEAccessibilityTextMarker EndMarker { get; set; }
+	}
+
+	[iOS (18, 2), TV (18, 2), Mac (15, 2), MacCatalyst (18, 2)]
+	[Protocol]
```
Comment (rolfbjarne) [inline src/browserenginekit.cs:1244]:
```suggestion
	[Protocol (BackwardsCompatibleCodeGeneration = false)]
```

File: src/browserenginekit.cs (hunk 3 above)
Comment (rolfbjarne) [reply src/browserenginekit.cs:1245]: Besides marking this protocol as not requiring backwards code gen, I'm not sure what else you suggest?

**Review by dalexsoto — APPROVED** [review]

**Pull request merged**